src/api-functions/open-ai-request/index.ts

```diff
@@ -16,12 +14,23 @@ import {
   SongType,
 } from "~/types";
 
-function getValidProps(type: ContentType, chunkedTextContent: Array<string>) {
+async function getValidProps(type: ContentType, chunkedTextContent: Array<string>, url?: string) {
   switch (type) {
     case "text":
     case "song":
     case "article":
       if (!chunkedTextContent || !chunkedTextContent.length) throw new Error("no data provided");
+      if (chunkedTextContent[0].split(" ").length < 200) {
```

Comment:
This should only throw for articles. A song could be less than 200 words, so can input text.

For example Coldplay's Talk is just 219 words.

Reply:
good catch!!